package.json

```diff
@@ -13,10 +13,11 @@
     "serve": "source .env && DEBUG=graphile-build-pg NODE_EXTRA_CA_CERTS=./db-ssl-certificate.pem npx ts-node ./src/serve.ts --cors",
     "connect-db-root": "source .env && PGPASSWORD=$POSTGRES_PASSWORD psql -d postgres -h $POSTGRES_HOST -p 5432 -U $POSTGRES_USERNAME",
     "connect-db": "source .env && PGPASSWORD=$POSTGRES_PASSWORD psql -d $POSTGRES_DATABASE -h $POSTGRES_HOST -p 5432 -U $POSTGRES_USERNAME",
-    "save-db": "source .env && PGPASSWORD=$POSTGRES_PASSWORD pg_dump -f ./db/db_dump -b -C -d $POSTGRES_DATABASE -h $POSTGRES_HOST -p 5432 -U $POSTGRES_USERNAME",
-    "restore-db": "source .env && PGPASSWORD=$POSTGRES_PASSWORD psql -h $POSTGRES_HOST -p 5432 -U $POSTGRES_USERNAME -d postgres < ./db/db_dump",
-    "drop-db": "read -p \"u sure? ctrl-c to abort \" -n 1 -r; source .env && PGPASSWORD=$POSTGRES_PASSWORD psql -d postgres -h $POSTGRES_HOST -p 5432 -U $POSTGRES_USERNAME -c \"drop database web3_music_pipeline;\"",
-    "reset-db": "yarn drop-db && yarn restore-db",
+    "save-db": "rm -rf db/dump/ && source .env && PGPASSWORD=$POSTGRES_PASSWORD pg_dump -F d -f ./db/dump -b -C -d $POSTGRES_DATABASE -h $POSTGRES_HOST -p 5432 -U $POSTGRES_USERNAME",
+    "drop-db": "read -p \"u sure? ctrl-c to abort \" -n 1 -r; source .env && PGPASSWORD=$POSTGRES_PASSWORD psql -d postgres -h $POSTGRES_HOST -p 5432 -U $POSTGRES_USERNAME -c \"drop database $POSTGRES_DATABASE;\"",
+    "restore-full-db": "source .env && PGPASSWORD=$POSTGRES_PASSWORD pg_restore ./db/dump  -h $POSTGRES_HOST -p 5432 -U $POSTGRES_USERNAME --create --clean --if-exists --verbose --verbose -F d -d postgres",
+    "reset-db": "yarn drop-db && yarn restore-full-db",
+    "restore-db-table": "source .env && PGPASSWORD=$POSTGRES_PASSWORD pg_restore ./db/dump  -h $POSTGRES_HOST -p 5432 -U $POSTGRES_USERNAME --verbose --verbose -F d -d $POSTGRES_DATABASE --data-only -t",
```

Comment:
There are double `--verbose` flags. I don't think it makes a difference but we can probably tweak it on the next pass.

Awesome new functionality! 🎉